cli/collection_create.go

```diff
@@ -66,35 +65,19 @@ Example: create from stdin
 				return ErrNoDocOrFile
 			}
 
-			var docMap any
-			if err := json.Unmarshal(docData, &docMap); err != nil {
-				return err
-			}
-
-			switch t := docMap.(type) {
-			case map[string]any:
-				doc, err := client.NewDocFromMap(t)
+			if client.IsJSONArray(docData) {
```

Comment:
praise: This is quite a lot nicer, thanks :)

Reply:
Thanks :)